app/views/content/shared/block-promos/_findtda_mailing_routes.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="row">
+  <section class="container">
+    <section class="image-blocks">
+      <%= render CallsToAction::FindBlockComponent.new(
+        title: "Find a teacher degree apprenticeship",
+        text: "Explore the different teacher degree apprenticeships available.",
+        link_target: "https://find-teacher-training-courses.service.gov.uk/results?applications_open=true&minimum_degree_required=no_degree_required"
```

Comment:
I wonder if we should change the link text too:

FROM `Find teacher training courses`
TO `Find apprenticeship courses`